### PR TITLE
add pagination to Teachers + update helpers for statuses and nativeLanguages

### PR DESCRIPTION
**Pull request by nondescryptid**
# Add paginations to Teachers.Vue + update helpers 

addresses #127 

Adds pagination to teachers table (10 teachers per page). 
Fixes helpers for nativelanguages and statuses by declaring variables (so they can be used locally), then exporting it. 
If these changes are not made, it is not possible to create new teachers/students 

## Test Steps

1. Use Postman 
2. Create a new Teacher/Students (sample request body: {"PhoneNumber":"93844322","FullName":"arjay","Source":"","Email":"arjay@gmail.com","NativeLanguageString":"CHINESE","SecondLanguageString":"TAMIL","LanguageProficiency":"Fluent","EnglishProficiency":"Intermediate","Notes":"","StatusString":"UNMATCHED"})
3. See if backend (api/teachers or api/students) is updated

### CODE REVIEW

**Review by therealyingtong — APPROVED** [review]
Untested approval

File: backend/src/helpers/nativeLanguages.js [hunk 1]
```diff
@@ -1,15 +1,17 @@
 const db = require("../models/nativeLanguage");
+const { get } = require("../routes/teachers");
```
Comment (therealyingtong) [inline backend/src/helpers/nativeLanguages.js:2]: is this used anywhere?

**Review by xuanjane — APPROVED** [review]
LGTM :)

File: backend/src/helpers/nativeLanguages.js (hunk 1 above)
Comment (nondescryptid) [reply backend/src/helpers/nativeLanguages.js:2]: Don't think so — I didn't type that but it somehow is being reflected as an addition. weird

**Pull request merged**